modules/base-laws/src/test/scala/io/janstenpickle/trace4cats/base/context/laws/LiftTests.scala

```diff
@@ -0,0 +1,5 @@
+package io.janstenpickle.trace4cats.base.context.laws
+
+class LiftTests extends BaseSuite {
+  checkAll("Lift[Option, Option]", discipline.LiftTests[Option, Option].lift[String, Int])
```

Comment:
`LiftTests` are already in `UnliftTests`. I think this file is redundant. Could you double check?

Reply:
I'm not following 100%. There are existing files named [`LiftTests.scala`](https://github.com/trace4cats/trace4cats/blob/master/modules/base-laws/src/main/scala/io/janstenpickle/trace4cats/base/context/laws/discipline/LiftTests.scala) and [`UnliftTests.scala`](https://github.com/trace4cats/trace4cats/blob/master/modules/base-laws/src/main/scala/io/janstenpickle/trace4cats/base/context/laws/discipline/UnliftTests.scala), but they're just traits for setting up discipline laws, not for actually checking the laws. I'm happy to move or rename this file, just let me know what you'd prefer it to be called.